README.md

```diff
@@ -30,34 +32,65 @@ If you use Maven, run the following command in a terminal window (in the complet
 ./mvnw spring-boot:run
 ```
 
-
 ### Run client
 
 1. Navigate into the client folder:
+
 ```
 cd client
 ```
 
 2. Install dependencies
+
 ```
 npm i
 ```
 
 3. Create a `.env` folder and add environment variables:
+
 ```
 REACT_APP_DESCOPE_PROJECT_ID="YOUR_DESCOPE_PROJECT_ID"
 ```
 
 4. Start the application
+
 ```
 npm start
 ```
 
->Note: If you're not running the client at http://localhost:3000 you may need to change the server's CrossOrigin domain to wherever you're hosting it (in JavaSampleAppApplication.java).
+>Note: If you're not running the client at <http://localhost:3000> you may need to change the server's CrossOrigin domain to wherever you're hosting it (in JavaSampleAppApplication.java).
+
 ```
 @CrossOrigin(origins = "http://localhost:3000")
 ```
 
+## Tenant-based OIDC SSO Setup
+
+You will need to configure a tenant in your Descope console with OIDC. Then, you can use the associated tenant ID to start SSO, redirect to the IdP authentication portal, and then exchange the returned code for 
+authenticated user info. We'll include the steps to set up in the UI here, but this can also be done via API or SDK.
+
+1. Create a tenant [here](https://app.descope.com/tenants)
+2. Enable [SSO via OIDC](https://app.descope.com/tenant/authentication/sso/tenant-a) and configure an Identity Provider
```

Comment:
Note that this link will work only if user have tenant with that id

Reply:
Adjusted to be instructions w/o link